src/Sarif.Multitool/ValidateCommand.cs

```diff
@@ -15,7 +15,7 @@ namespace Microsoft.CodeAnalysis.Sarif.Multitool
 {
     internal class ValidateCommand : AnalyzeCommandBase<SarifValidationContext, ValidateOptions>
     {
-        public override string Prerelease => VersionConstants.Prerelease;
+        public override string Prerelease => string.Empty;
```

Comment:
>Prerelease => string.Empty; [](start = 31, length = 27)

I am unsure about the purpose of the "prerelease" string, for now i have updated the value from the actual build variable (which was an empty string) to surgically break the dependency. Should we simply remove this variable usage now?

Reply:
yes, you're right, remove it.

---
In reply to: [282281902](https://github.com/microsoft/sarif-sdk/pull/1484#discussion_r282281902) [](ancestors = 282281902)